src/components/user-profile-page/_user-profile-page.scss

```diff
@@ -0,0 +1,314 @@
+@use './components/global/mixins';
+
+.user-profile-page {
+  color: #373738;
+  font-family: arial;
+  height: 100%;
+  position: relative;
+
+  &__header {
+    background: #ffd500;
+    padding: 11rem 0 16rem;
+    position: relative;
+
+    img {
+      display: block;
+      margin: auto;
+      padding: 0 2.1rem;
+      width: 85%;
+    }
+  }
+
+  &__header__title {
+    font-size: 4rem;
+    padding: 7rem 3rem 0rem;
+    text-align: center;
+  }
+
+  &__header__description {
+    font-size: 2.5rem;
+    padding: 2rem 5rem 8rem;
+    text-align: center;
+  }
+
+  &__header__arrow-button {
+    background: #383d3f url("/components/user-profile-page/images/arrow.svg") no-repeat center center;
+    border-radius: 50%;
+    cursor: pointer;
+    height: 60px;
```

Comment:
@3aPretorius could you change the px values to use rem instead? e.g. 6rem in this instance